.circleci/config.yml

```diff
@@ -8,8 +8,52 @@
 version: 2
 jobs:
 
-  # job: Mac OS/X raw ("build-and-test")
-  build-and-test:
+  # job: Mac OS/X raw ("build-debug")
+  build-debug:
+    macos:
+      xcode: '10.1.0'
+    working_directory: ~/numenta/htm.core
+    parallelism: 1
+    environment:
+      XCODE_SCHEME: htm
+      XCODE_WORKSPACE: htm
+      ARCHFLAGS: -arch x86_64
+    steps:
```

Comment:
There's a lot of overlap for the 2 build jobs, would it be possible to make the settings common, and just disambiguate the Release/Debug modes? (if not, it's ok)